sklearn/linear_model/logistic.py

```diff
@@ -610,7 +610,7 @@ def logistic_regression_path(X, y, pos_class=None, Cs=10, fit_intercept=True,
                                  "solver cannot handle multiclass with "
                                  "class_weight of type dict. Use the lbfgs, "
                                  "newton-cg or sag solvers or set "
-                                 "class_weight='auto'")
+                                 "class_weight='balanced'")
```

Comment:
It's kind of unrelated to what you are fixing, I'm not sure if we should address it in this PR or not. It just seems odd.